packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesUtils.kt

```diff
@@ -130,16 +128,15 @@ object UpdatesUtils {
       if (expectedBase64URLEncodedHash != null && expectedBase64URLEncodedHash != hashBase64String) {
         throw IOException("File download was successful but base64url-encoded SHA-256 did not match expected; expected: $expectedBase64URLEncodedHash; actual: $hashBase64String")
       }
-
       // only rename after the hash has been verified
       // Since renameTo() does not expose detailed errors, and can fail if source and destination
       // are not on the same mount point, we do a copyTo followed by delete
+      // ENG-10888: if there are two assets with identical content, they will be written to the same file path,
+      // so we allow overwrites
```

Comment:
```suggestion
      // if there are two assets with identical content, they will be written to the same file path,
      // so we allow overwrites
```

(no need to include the task unless there's something ongoing and this is more of a TODO. blame should be sufficient to trace through to the task)